.ci/setup.sh

```diff
@@ -128,8 +126,19 @@ install_extra_tools() {
 }
 
 main() {
-	bash -f "${cidir}/install_go.sh" -p -f
-	setup_distro_env
+	local setup_type="default"
+
+	# Travis only needs a very basic setup
+	set +o nounset
+	[ "$TRAVIS" = "true" ] && setup_type="minimal"
+	set -o nounset
+
+	[ "$setup_type" = "default" ] && bash -f "${cidir}/install_go.sh" -p -f
```

Comment:
@chavafg - I was wondering about whether we should run this under Travis? Currently, we're relying on the following from `.travis.yml`:

```yaml
go:
  - "1.11.x"
  - stable
```

The advantage of using the travis config to specify is that we can test on > 1 version of go. But if we used `install_go.sh`, we could simplify the Travis config further and we wouldn't need to update all the Travis config files when we moved to a new version of golang. Any thoughts?

Reply:
I think we can use `install_go.sh` under travis, although I am not sure if it will work as it is... Maybe we can try on an upcoming PR. This way we don't need to hardcode the `golang` version on the `.travis.yml` file.

Reply:
Sounds good. Ok - I'll leave this code "as-is" for now.

Reply:
Issue #1738 raised for this.